src/doc/rustc/src/platform-support/loongarch-linux.md

```diff
@@ -0,0 +1,87 @@
+
+The target name follow this format: `<machine>-<vendor>-<os><fabi_suffix><abiext_suffix>`,where `<machine>` specifies the CPU family/model, `<vendor>` specifies the vendor and `<os>` the operating system name.
+While the integer base ABI is implied by the machine field, the floating-point base ABI and the ABI extension type are encoded with two string suffices (`<fabi-suffix>``<abiext-suffix>`) to the os field of the specifier, respectively.
+
+|    `<fabi-suffix>`     |                           `Description`                            |
+|------------------------|--------------------------------------------------------------------|
+|          f64           | The base ABI use 64-bits FPRs for parameter passing.(lp64d, ilp32d)|
+|          f32           | The base ABI uses 32-bit FPRs for parameter passing. (lp64f ilp32f)|
+|          sf            | The base ABI uses no FPR for parameter passing. (lp64s ilp32s)     |
+
+|   `<abiext-suffix>`    |                       `ABI extension type`                         |
+|------------------------|--------------------------------------------------------------------|
+|    (empty string)      |                               base                                 |
+
+|`ABI type(Base ABI/ABI extension)`| `C library` | `kernel` |          `target tuple`          |
+|----------------------------------|-------------|----------|----------------------------------|
+|           lp64d/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnuf64 |
+|           lp64f/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnuf32 |
+|           lp64s/base             |   glibc     |  linux   | loongarch64-unknown-linux-gnusf  |
+|           ilp32d/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnuf64 |
+|           ilp32f/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnuf32 |
+|           ilp32s/base            |   glibc     |  linux   | loongarch32-unknown-linux-gnusf  |
+|           lp64d/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslf64|
+|           lp64f/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslf32|
+|           lp64s/base             |  musl libc  |  linux   | loongarch64-unknown-linux-muslsf |
+|           ilp32d/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslf64|
+|           ilp32f/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslf32|
+|           ilp32s/base            |  musl libc  |  linux   | loongarch32-unknown-linux-muslsf |
+
+## Target maintainers
+
+- zhaixiaojuan, `zhaixiaojuan@loongson.cn`, https://github.com/zhaixiaojuan
+
+## Requirements
+
+This target is cross-compiled.
+A platform-provided C compiler toolchain is required, and obtaining address of the cross toolchain [GNU LoongArch toolchain](https://github.com/loongson/build-tools/releases).
```

Comment:
```suggestion
A GNU toolchain for LoongArch target is required.  It can be downloaded from https://github.com/loongson/build-tools/releases, or built from the source code of GCC (12.1.0 or later) and Binutils (2.39 or later).
```

I'm not really sure about how to specify the minimal Binutils version, as 2.39 not released yet but 2.38 does not work.